### DISCUSSION TITLE
how to exclude specific phrases from highlights?

### DISCUSSION THREAD

**Discussion by of-me-is** [Chatterino/chatterino2#5505] [Q&A] [answered]
for example, i want to highlight messages with "right", but not the ones with "go right". is there an option to exclude specifically "go right" from highlights, similarly to how you can exclude something from search results when browsing by doing -"blah blah"?

so,
"i would go right" would NOT be highlighted
"random phrase" would NOT be highlighted
"you're right" would be highlighted

**Comment by Mm2PL** [comment]
Use a regular expression. https://wiki.chatterino.com/Regex/ 

For search you can just use the syntax: `regex:"YOUR REGULAR EXPRESSION HERE"` and for highlights you check the box.

**Reply by of-me-is** [reply]
maybe i'm doing it wrong, but as far as i understand, there's only an option to highlight everything except a specific phrase, and i want to just exclude one phrase instead, not highlight everything else

**Reply by pajlada** [reply, marked as answer]
As an example, this would be a regex you can use to match/not match the phrases you mentioned above https://regex101.com/r/I4AO8A/1
